jumanji/training/train.py

```diff
@@ -103,6 +105,7 @@ def epoch_fn(training_state: TrainingState) -> Tuple[TrainingState, Dict]:
             # Training
             with train_timer:
                 training_state, metrics = epoch_fn(training_state)
+                jax.block_until_ready((training_state, metrics))
```

Comment:
Remind me why we are adding this?

Reply:
I imagine it is just to make sure there is no async_dispatch calls coming back and that the epoch has actually finished when we call it, otherwise we might get misleading metrics

Reply:
It's so the timing is correct, indeed.

Reply:
Makes sense 👍